newIDE/app/src/GameDashboard/GameCard.js

```diff
@@ -1,92 +1,275 @@
-                          label={
-                            game.discoverable ? (
-                              <Trans>Discoverable on Liluo.io</Trans>
-                            ) : (
-                              <Trans>Published on Liluo.io</Trans>
-                            )
-                          }
+                        <RaisedButton
+                          label={<Trans>Open in browser</Trans>}
+                          onClick={openGameUrl}
+                          primary
+                          disabled={!game.publicWebBuildId}
                         />
-                      </>
-                    )}
-                  </Line>
-                }
-              />
-              <CardActions>
-                <ResponsiveLineStackLayout
-                  expand
-                  noMargin
-                  justifyContent="flex-end"
-                >
-                  {game.publicWebBuildId && (
-                    <RaisedButton
-                      label={<Trans>Open in browser</Trans>}
-                      onClick={openGameUrl}
-                    />
-                  )}
-                  <RaisedButton
-                    label={<Trans>Manage game</Trans>}
-                    onClick={onOpenGameManager}
-                    primary
-                  />
-                </ResponsiveLineStackLayout>
-              </CardActions>
-            </Column>
-          </ResponsiveLineStackLayout>
-        </Card>
+                        <IconButton
+                          size="small"
+                          disabled={!game.publicWebBuildId}
+                          onClick={() => setShowShareDialog(true)}
+                          tooltip={t`Share`}
+                        >
+                          <ShareIcon />
+                        </IconButton>
+                      </ResponsiveLineStackLayout>
+                    </Column>
+                  </ResponsiveLineStackLayout>
+                  <ResponsiveLineStackLayout
+                    noMargin
+                    justifyContent="flex-start"
+                  >
+                    <Column expand alignItems="flex-start" noMargin>
+                      <Toggle
+                        labelPosition="left"
+                        onToggle={onToggleDiscoverable}
+                        toggled={getCurrentState(
+                          game.discoverable,
+                          isDiscoverable
+                        )}
+                        label={<Trans>Make discoverable on Liluo</Trans>}
+                      />
+                      <Toggle
+                        labelPosition="left"
+                        onToggle={onToggleAcceptsGameComments}
+                        toggled={getCurrentState(
+                          game.acceptsGameComments,
+                          acceptsGameComments
+                        )}
+                        label={<Trans>Open for feedback on Liluo</Trans>}
```

Comment:
I feel that we should improve this wording, it's too similar to the one for builds.
Should it be `Show feedback banner on Liluo.io game page`?
And for builds: `Ask for feedback on all build pages`